portfolio/src/main/webapp/index.html

```diff
@@ -11,11 +11,11 @@
     <script>
         $(document).ready(function () {
             // Hide the div
-            $("#img_delay").hide();
+            //$("#img_delay").hide();
```

Comment:
When you are done experimenting, you will want to delete the commented-out sections.
But during development, this is fine.